tools/clang/test/CodeGenHLSL/uavBarrier.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxc -E main -T ps_6_0 %s
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
```

Comment:
This test already exists under HLSLFileCheck/validation, so it should just be deleted from here.  This one is referenced in ValidatorTest.cpp, but that's redundant, so that test group should be removed as well.  Incidentally, the CHECK message here doesn't check for anything in particular, just succeeds if the compilation succeeded, I suppose.  It would be nice if it at least did `// CHECK: define void @main()` like what some other tests do when just verifying successful compilation.